### PR TITLE
Register and login view

### PR DESCRIPTION
**Pull request by KarinaFernandez**
- A new storyboard (Register) has been added.
- A new view controller with two buttons to change between Register and Login has been added. 
- The "form" is reused for the login and register option. Be default the register option is selected, so if the user doesn't tapped in login, is assumed that is registering (appear register in green) and is checked that the values are validated (email, valid format and password with length of 6 with 1 letter and 1 upper case). 
By the other way when the user tap in login is going to validate if the credentials are valid.
Actually user database doesn't exist, so no user is created and no validated if the credentials are ok.

**Related cards:**
#6
#7 

**UI preview:**
login register view

### CODE REVIEW

File: DayDreaming/DayDreaming/Controllers/RegisterViewController.swift [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    
+    func goBack() {
+        _ = navigationController?.popViewController(animated: true)
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+     
+        emailTextField.attributedPlaceholder = NSAttributedString(string: "Email", attributes: [NSForegroundColorAttributeName: UIColor.dydPinkishGrey])
+        passwordTextField.attributedPlaceholder = NSAttributedString(string: "Password", attributes: [NSForegroundColorAttributeName: UIColor.dydPinkishGrey])
```
Comment (santiagofm) [inline DayDreaming/DayDreaming/Controllers/RegisterViewController.swift]: please move this configs to a separate function and call it from `viewDidLoad`

File: DayDreaming/DayDreaming/Controllers/RegisterViewController.swift [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    // MARL: View did load
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Add navigation controller
+        self.navigationController?.setNavigationBarHidden(false, animated: true)
+        // Set navigation controller transparent
+        self.navigationController?.navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
+        self.navigationController?.navigationBar.shadowImage = UIImage()
+        self.navigationController?.navigationBar.isTranslucent = true
```
Comment (santiagofm) [inline DayDreaming/DayDreaming/Controllers/RegisterViewController.swift]: please move all this configs to a separate function

File: DayDreaming/DayDreaming/Controllers/RegisterViewController.swift [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        loginButton.isSelected = false
+    }
+    
+    @IBAction func loginTapped(_ sender: UIButton) {
+        loginButton.isSelected = true
+        registerButton.isSelected = false
+    }
+    
+    @IBAction func registerLoginTapped(_ sender: UIButton) {
+        messageLabel.isHidden = false
```
Comment (mcousillas6) [inline DayDreaming/DayDreaming/Controllers/RegisterViewController.swift]: This method is kind of long, you could refactor it if you separate the register and login functionality.

File: DayDreaming/DayDreaming/Controllers/RegisterViewController.swift [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    func validCredentials (email: String, password: String) -> Bool {
+        let valid = false
+        // Validate the email - password in user DB
+        return valid
+    }
+    
+    // MARK: Validate password
+    func passwordValid(password: String) -> Bool {
+        var valid = false
+        if !password.isEmpty {
```
Comment (mcousillas6) [inline DayDreaming/DayDreaming/Controllers/RegisterViewController.swift]: I think you can refactor this to 
`guard 
     !password.isEmpty 
     && password.characters.count >= 6 
     && password.validateCapital() 
     && password.validateNumber() 
else { return false }
return true`

**Pull request merged**